packages/@react-spectrum/label/chromatic/StaticField.chromatic.tsx

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {CalendarDate, CalendarDateTime, Time, ZonedDateTime} from '@internationalized/date';
+import {Meta, Story} from '@storybook/react';
+import React from 'react';
+import type {SpectrumStaticFieldProps, SpectrumStaticFieldTypes} from '../src/StaticField';
+import {StaticField} from '../';
+
+const meta: Meta<SpectrumStaticFieldProps<SpectrumStaticFieldTypes>> = {
+  title: 'StaticField',
+  component: StaticField
+};
+
+export default meta;
+
+const Template = (): Story<SpectrumStaticFieldProps<SpectrumStaticFieldTypes>> => (args) => (
```

Comment:
if you have some time, it'd be great if we could write our stories in CSF 3.0, no worries if you don't have the time though
See https://github.com/adobe/react-spectrum/pull/3129/files for an example